styles/style.css

```diff
@@ -0,0 +1,214 @@
+/* BASE RULES & OVERRIDES FOR DEFAULT BROWSER STYLES*/
+html {
+  font-size: 80%;
+  height: 100%;
+}
+
+* {
+  font-family: 'Lato', Verdana, Helvetica, sans-serif;
+  line-height: 1rem;
+  margin: 0;
+  padding: 0;
+  color: DimGrey;
+}
+
+body {
+  height: 100%;
+}
+
+.override article { padding: 1rem .5rem; margin: 1rem auto; }
+.override h1 { font-size: 4.8rem; line-height: 1.4; text-align: center; }
+.override h2 { font-size: 2.4rem; line-height: 1.4; }
+.override h3 { font-size: 2.0rem; line-height: 1.4; font-family: 'Zilla Slab', serif; padding-bottom: .25rem; }
+.override h4 { font-size: 1.6rem; line-height: 1.4; font-style: italic; }
+.override p, cite, li { font-size: 1.6rem; line-height: 1.4; padding-bottom: 1rem; }
+.override ul { text-decoration: none; }
+.override a, a:hover, a:visited, a:active, a:link { text-decoration: none; color: Teal; }
+.override a:hover { color: DarkCyan; }
+.override img { width: 100%; }
```

Comment:
You actually shouldn't need the override class at all. I removed it locally just to test it out and it looks like these styles are still applied without the override class